fix publish process

Currently if you have nectarine installed you get: `Error: Cannot find module 'nectarine'`

* add prepublish script to build
* add `main` to package.json
* add `files` to package.json to remove development files

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "version": "0.0.2",
   "scripts": {
     "build": "webpack",
-    "prepblish": "npm run build",
+    "prepublish": "npm run build",
     "test": "mocha 'src/**/*.spec.ls'"
   },
   "license": "MIT",

